src/Microsoft.OData.Core/ODataWriterCore.cs

```diff
@@ -2451,7 +2451,7 @@ private void InterceptException<TArg0>(Action<ODataWriterCore, TArg0> action, TA
         /// Make sure to only use anonymous functions that don't capture state from the enclosing context, 
         /// so the compiler optimizes the code to avoid delegate and closure allocations on every call to this method.
         /// </remarks>
-        private async Task InterceptExceptionAsync(Func<ODataWriterCore, Task> action)
+        private async Task InterceptExceptionAsync(Func<ODataWriterCore, Task> action, ODataItem currentScopeItem)

```

Comment:
Add a doc comment for the second parameter.

Reply:
Thanks for this. It was in my mind at one point then I forgot